### PR TITLE
[disaster dashboard] initial page set up

### PR DESCRIPTION
**Pull request by chejennifer**
set up skeleton for the new disaster dashboard page (so future PRs can be submitted in parallel)

This is one part of the changes for the disaster dashboard MVP here on dev:
https://dev.datacommons.org/dev/disaster-dashboard

### CODE REVIEW

File: server/routes/dev.py [hunk 1]
```diff
@@ -36,4 +36,12 @@ def screenshot(folder):
     if os.environ.get('FLASK_ENV') == 'production':
         flask.abort(404)
     images = list_png(SCREENSHOT_BUCKET, folder)
-    return flask.render_template('dev/screenshot.html', images=images)
\ No newline at end of file
+    return flask.render_template('dev/screenshot.html', images=images)
+
+
+@bp.route('/disaster-dashboard')
+def disaster_dashboard():
```
Comment (shifucun) [inline server/routes/dev.py:43]: nit: make `os.environ.get('FLASK_ENV')` a module level var

**Review by shifucun — APPROVED** [review]

File: server/routes/dev.py (hunk 1 above)
Comment (beets) [reply server/routes/dev.py:43]: alternatively, use 
```
not os.environ.get('FLASK_ENV') in ['autopush', 'local', 'dev]
```

File: static/js/disaster_dashboard/types.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+ * limitations under the License.
+ */
+
+import { MapPoint } from "../chart/types";
+
+/**
+ * Types specific to Disaster Dashboard
+ */
+
+export enum DisasterType {
```
Comment (beets) [inline static/js/disaster_dashboard/types.ts:23]: hm, this is technically a constant so should go in constants.ts

**Review by beets — APPROVED** [review]
thanks for splitting this up into smaller pr's

File: server/routes/dev.py (hunk 1 above)
Comment (chejennifer) [reply server/routes/dev.py:43]: done

File: static/js/disaster_dashboard/types.ts (hunk 2 above)
Comment (chejennifer) [reply static/js/disaster_dashboard/types.ts]: done

**Comment by chejennifer** [comment]
thanks for the reviews!

**Pull request merged**